db/internal_stats.cc

```diff
@@ -1056,6 +1063,63 @@ void InternalStats::DumpDBStats(std::string* value) {
   db_stats_snapshot_.write_stall_micros = write_stall_micros;
 }
 
+void InternalStats::DumpDBMapStats(
```

Comment:
Is the new function used in this PR? If not, I suggest we remove it from the PR to avoid confusion.